src/templates/partials/_header.html

```diff
@@ -21,7 +21,7 @@ <h1 class="logo">
     <script>
 
         $(window).scroll(function () {
-            if ($(this).scrollTop() > 100) {
+            if ($(this).scrollTop() > 30) {
```

Comment:
It will be cool, if you add a comment here about where do you take this 30 pixels